packages/react-dom/src/__tests__/ReactDOMServerIntegrationElements-test.js

```diff
@@ -255,8 +255,7 @@ describe('ReactDOMServerIntegration', () => {
           <div>
             {[['a'], 'b']}
             <div>
```

Comment:
strange if prettier did this, since the above patterns have more of a `new line` pattern... like
```
<a href=''>
   span
</a>
```